packages/now-build-utils/test/unit.test.js

```diff
@@ -475,19 +475,71 @@ it('Test `detectBuilders`', async () => {
     expect(builders[0]).toEqual({
       src: 'api/teams/members.ts',
       use: '@now/node',
-      config: { zeroConfig: true, memory: 128, maxDuration: 10 },
+      config: { zeroConfig: true, functions },
     });
     expect(builders[1]).toEqual({
       src: 'api/users/[id].ts',
-      use: 'my-custom-runtime-package',
-      config: { zeroConfig: true },
+      use: 'my-custom-runtime-package@1.0.0',
+      config: { zeroConfig: true, functions },
     });
     expect(builders[2]).toEqual({
       src: 'package.json',
-      use: '@now/next@canary',
-      config: { zeroConfig: true, memory: 3008 },
+      use: '@now/next@1.0.0-canary',
+      config: { zeroConfig: true, functions },
     });
   }
+
+  {
+    // invalid function memory
```

Comment:
```suggestion
    // invalid function key
```

I'm guessing this has something to do with the key and not the memory?